tests/x11regressions/evolution/evolution_mail_ews.pm

```diff
@@ -0,0 +1,128 @@
+#Test Case #1503965: Evolution: Setup MS Exchange account
+use strict;
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $mailbox     = 'zzzSUSEExTest19@microfocus.com';
+    my $mail_passwd = 'P@$$w0rd2015';
+
+    mouse_hide(1);
+
+    # Clean and Start Evolution
+    x11_start_program("xterm -e \"killall -9 evolution; find ~ -name evolution | xargs rm -rf;\"");
+    x11_start_program("evolution");
+    if (check_screen "evolution-default-client-ask") {
+        assert_and_click "evolution-default-client-agree";
+    }
+
+    # Follow the wizard to setup mail account
+    assert_screen "test-evolution-1";
```

Comment:
the following should work faster as the test would not need to wait for timeout on check_screen:
```
assert_screen([qw/evolution-default-client-ask test-evolution-1/]);
if (match_has_tag('evolution-default-client-ask')) {
    assert_and_click 'evolution-default-client-agree';
    assert_screen 'test-evolution-1';
}
```

and the same some lines down but both approaches work. only change it if you like it.